apps/escrow/frontend/src/systems/Core/pages/CreateWallet.tsx

```diff
@@ -0,0 +1,42 @@
+import { useNavigate } from "react-router-dom";
+import { Layout } from "../components/Layout";
+import { useAppContext } from "../context/AppContext";
+import { Pages } from "../types/pages";
+import { Card, Button } from "@fuels-ui/react";
+
+
+export default function CreateWallet() {
+    const { createWallet } = useAppContext()!;
+    const navigate = useNavigate();
+
+    // const createWalletMutation = useMutation(async () => createWallet, {
+    //     onSuccess: () => {
+    //         navigate(Pages.escrow);
+    //     },
+    // });
+
+    function handleCreateWallet() {
+        //createWalletMutation.mutate();
+        createWallet();
+        navigate(Pages.escrow);
+    }
+
+    return (
```

Comment:
Instead of a create wallet page it would be better to jump straight into the escrow page and have an option to create as many escrows as we like (depending on the balance of each wallet).

Then we could switch between each wallet and have it render whatever escrows that wallet has deployed.
Moreover, we could have a drop down similar to what [remix](https://remix.ethereum.org/) has with its addresses. This way the user can easily switch between a set of predetermined addresses that already have some balance.

Reply:
That makes sense, we can just create and fund the wallets for the user.

I agree that we should be able to switch between wallets and render that wallets deployed escrows.